src/sentry/static/sentry/app/views/eventsV2/querycard.tsx

```diff
@@ -69,6 +59,20 @@ class QueryCard extends React.PureComponent<Props> {
   }
 }
 
+const QueryCardContent = styled('div')`
+  flex-grow: 1;
+  overflow: hidden;
+  margin-right: ${space(1)};
+`;
+
+const Avatar = styled('div')`
```

Comment:
Is this going to contain some user related content soon?

Reply:
Yep! Just trying to work with Will to get it 👍